src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,438 @@
+package Duke;
+
+import Duke.Exceptions.InvalidCommandException;
+import Duke.Exceptions.InvalidFormatException;
+import Duke.Tasks.Deadline;
+import Duke.Tasks.Event;
+import Duke.Tasks.Task;
+import Duke.Tasks.Todo;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
+    public static void main(String[] args) {
```

Comment:
Keep main method clean. Shift details of implementation to separate functions in appropriate classes